jobs/webcompat-kb/webcompat_kb/crux.py

```diff
@@ -116,6 +116,47 @@ def update_sightline_data(client: bigquery.Client, config: Config, date: int) ->
         logging.info(f"Sightline data has {result.total_rows} rows")
 
 
+def update_min_rank_data(client: bigquery.Client, config: Config, date: int) -> None:
+    if config.write:
+        insert_str = f"INSERT `{config.bq_project}.{config.bq_crux_dataset}.host_min_ranks` (yyyymm, host, global_rank, local_rank, sightline_rank)"
+    else:
+        insert_str = ""
+
+    query = f"""
+{insert_str}
+SELECT
+  {date} as yyyymm,
+  NET.HOST(origin) AS host,
+  MIN(
+  IF
+    (origin_ranks.country_code = "global", origin_ranks.rank, NULL)) AS global_rank,
```

Comment:
Can we put these all in one line, as in

```
MIN(IF(origin_ranks.country_code = "global", origin_ranks.rank, NULL)) AS global_rank,
```
?

That would make this a bit easier to understand

Reply:
I also kind of hate it, but that's what the Big Query formatter produces, so I'm tempted to leave it.

Reply:
If an autoformatter did that, I'll concede my point.